docs/data/gridftp.md

```diff
@@ -111,6 +111,27 @@ All other configuration settings should be suitable for most purposes. However,
 
 ### Optional configuration
 
+#### Setting transfer limits
+
+Create file named `/etc/sysconfig/gridftp-limits`. To set a limit on the total or per-user number of transfers, set the following configuration:
```

Comment:
Hrm that still reads kinda strangely, how about:

> To set a limit on the total or per-user number of transfers, create a file named `/etc/sysconfig/gridftp-limits` with the following configuration:

Does creating `/etc/sysconfig/gridftp-limits` actually work? Is gridftp smart enough to pick up *any* `/etc/sysconfig-gridftp*` file?

Reply:
Actually I will have to do some testing on this.

Reply:
@brianhlin I checked and this limits were only coded for gridftp-hdfs in [here](https://github.com/bbockelm/gridftp-hdfs/commit/9ae16298579ff7f20673171c56e213bcf4cf305a) so it would make sense to add this here for only HDFS.